spec/site/connect_spec.rb

```diff
@@ -1,14 +1,14 @@
 RSpec.describe "RSMP site connection" do
 
-  def check_connection_sequence version, expected
+  def check_connection_sequence version, Expected
```

Comment:
what's up with the capitalizatin here?

Reply:
Woops, I managed to search-replace a tad more than I aimed for. Fixed it.

Reply:
alright :-)